tests/python/unit/test_sentence_splitter.py

```diff
@@ -15,4 +15,30 @@ def test_split_sentences_nan_values():
     text_df = pd.DataFrame({"text": [float("nan")]})
     tagger._initialize_tokenizer([tagger.language])
     text_df, text_column_tokenized = tagger._sentence_splitting(text_df, "text")
-    assert text_df[text_column_tokenized].iloc[0] == []
\ No newline at end of file
+    assert text_df[text_column_tokenized].iloc[0] == []
+
+
+def test_split_sentences_linebreaks():
+    """Test behavior when splitting sentences with linebreaks in the text column"""
+    ontology_df = pd.DataFrame({"tag": ["first", "second", "third"]})
+    tagger = Tagger(
+        ontology_df=ontology_df,
+        tag_column="tag",
+        category_column="None",
+        keyword_column="tag",
+        language="en",
+    )
+    text_df = pd.DataFrame(
+        {
+            "text": [
+                """first line with carriage return"""+"\r"+"""second line with two linebreaks
```

Comment:
Nitpick: in Python, the string
```
"""first line with carriage return"""+"\r"+"""second line with two linebreaks

third line with parenthesis)
Last line"""
```
Is exactly the same as 
```
'first line with carriage return\rsecond line with two linebreaks\n\nthird line with parenthesis)\nLast line'
```

I have a small preference for the second option, as it is more compact as shows which line break character is used

Reply:
Alright I have updated it here as you suggested 8276993